app/views/test_groups/edit.slim

```diff
@@ -0,0 +1,8 @@
+= render 'layouts/form' do
+  .new-form
+    .new-form--content
+      .new-form--content--title
+        h1 Editing test group
+      .new-form--content--form
+        = render 'form', test_group: @test_group
+        .new-form--content--form--image.test-img
```

Comment:
Why do we need that div here?

Reply:
It has an illustration as background-image, all the illustrations are done with background-image

![Screenshot 2020-09-14 at 17 19 15](https://user-images.githubusercontent.com/23219057/93097573-6e178900-f6ae-11ea-9271-b35157a21519.png)